core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -500,6 +521,15 @@ FileInStream openFile(AlluxioURI path)
   FileInStream openFile(AlluxioURI path, OpenFilePOptions options)
       throws FileDoesNotExistException, IOException, AlluxioException;
 
+  /**
+   * Schedules the given path to be asynchronously persisted to the under file system. If a
+   * synchronous persist operation is desired please see
+   * {@link FileSystemUtils#waitPersisted(FileSystem, AlluxioURI)}
+   *
+   * @param uri the uri of the file to persist
+   */
+  void persist(AlluxioURI uri) throws IOException;
```

Comment:
even if there aren't any options now, it's still good to include an options argument so that we can add options in the future without breaking the API

Reply:
good call. Will add. Didn't realize there's some options available already: `ScheduleAsyncPersistencePOptions`